boards/opentitan/README.md

```diff
@@ -0,0 +1,29 @@
+OpenTitan RISC-V Board
+=================
+
+- https://opentitan.org/
+
+OpenTitan is the first open source project building a transparent, high-quality reference design and integration guidelines for silicon root of trust (RoT) chips.
+
+You can get started with OpenTitan using either the Nexys Video FPGA board or simulation. See the OpenTitan [getting started](https://github.com/lowRISC/opentitan/blob/master/doc/ug/getting_started.md) for more details.
+
+Programming
+-----------
+
+Currently `make flash` is not supported and you will need to use the [OpenTitan development flow](https://github.com/lowRISC/opentitan/blob/master/doc/ug/getting_started_fpga.md#testing-the-demo-design) to flash the image.
+
+Hopefully in the future this can be improved.
+
+First setup the development board using the steps here: https://github.com/lowRISC/opentitan/blob/master/doc/ug/getting_started_fpga.md. You need to make sure the boot ROM is working and that your machine can communicate with the OpenTitan ROM. You will need to use the `prog` USB port on the board for this.
+
+In the OpenTitan repo build the `spiflash` program.
+
+```
+make -C sw/host/spiflash clean all
+```
+
+Then use the `spiflash` program to download the image to the board.
+
+```
+./sw/host/spiflash/spiflash --input=./target/riscv32imc-unknown-none-elf/release/opentitan.bin
+```
```

Comment:
Add a reminder that it operates at 230400 not 115200, and mention how to figure out which USB to connect to. Alternatively, link to the OT documentation that tells you how to connect to the serial port. Don't expect the reader to figure out to go back to the OT documentation.

Show what successful output looks like.
```
bootstrap: DONE!
Jump!
OpenTitan initialisation complete. Entering main loop
```